include/multipass/virtual_machine.h

```diff
@@ -85,8 +85,8 @@ class VirtualMachine : private DisabledCopyMove
     virtual void resize_memory(const MemorySize& new_size) = 0;
     virtual void resize_disk(const MemorySize& new_size) = 0;
     virtual void add_network_interface(int index, const NetworkInterface& net) = 0;
-    virtual void add_extra_interfaces_to_cloud_init(const std::string& default_mac_addr,
-                                                    const std::vector<NetworkInterface>& extra_interfaces) = 0;
+    virtual void apply_extra_interfaces_to_cloud_init(const std::string& default_mac_addr,
+                                                      const std::vector<NetworkInterface>& extra_interfaces) = 0;
```

Comment:
Name prefix changed from add to apply which better reflects what the function is doing.